src/Avalonia.Controls.TreeDataGrid/Avalonia.Controls.TreeDataGrid.csproj

```diff
@@ -8,6 +8,7 @@
     <PackageId>Avalonia.Controls.TreeDataGrid</PackageId>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Avalonia" Version="$(AvaloniaVersion)"/>
+    <PackageReference Include="Avalonia" Version="$(AvaloniaVersion)" />
+    <PackageReference Include="System.Reactive" Version="5.0.0" />
```

Comment:
Avalonia no longer includes this package?

Reply:
No, and it probably should be removed from the TreeDataGrid as well.
@jmacato if you need help with that, ping me.